scripts/remove_edge_associations.py

```diff
@@ -0,0 +1,85 @@
+import boto3
+from botocore.exceptions import ClientError
+import time
+
+def log(message):
+    print(message, flush=True)
+
+def detach_lambda_edge_associations(distribution_id):
+    try:
+        client = boto3.client('cloudfront')
+        
+        response = client.get_distribution_config(Id=distribution_id)
+        config = response['DistributionConfig']
+        etag = response['ETag']
+
+        behaviors = []
```

Comment:
Missing type e.g. `behaviours: list[str] = []` just so it's clear what we expect this array to hold